src/validation.cpp

```diff
@@ -492,15 +538,23 @@ class MemPoolAccept
     // only tests that are fast should be done here (to avoid CPU DoS).
     bool PreChecks(ATMPArgs& args, Workspace& ws) EXCLUSIVE_LOCKS_REQUIRED(cs_main, m_pool.cs);
 
+    // Run checks for mempool replace-by-fee.
+    bool ReplacementChecks(Workspace& ws) EXCLUSIVE_LOCKS_REQUIRED(cs_main, m_pool.cs);
+
+    // Enforce package mempool ancestor/descendant limits.
```

Comment:
Note the comment above `PreChecks` also mention evaluation package limits. I think both of statements are true, though they don't refer to the same set of package limits. IIRC the first ones in `PreChecks` are the in-mempool package limits while the second ones concern atomically accepted package, which is stricter under the union rule. 

I think we can improve documentation to establish clearly that we have two set of package limits to consider.

Reply:
Edited the comment to clarify that there's individual and package ancestor/desc limits